core/templates/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -506,6 +511,11 @@ angular.module('oppia').directive('questionsList', [
             var misconceptionsBySkill = ctrl.misconceptionsBySkill;
             var associatedSkillSummaries = ctrl.associatedSkillSummaries;
             var newQuestionIsBeingCreated = ctrl.newQuestionIsBeingCreated;
+            var categorizedSkills = null;
+            TopicsAndSkillsDashboardBackendApiService.fetchDashboardData().then(
+              function(response) {
+                categorizedSkills = response.data.categorized_skills_dict;
+              });
```

Comment:
I am a bit hesitant about including the TopicsAndSkillsDashboardApi in the questions-list directive just to fetch categorisedSkills data. Can we instead send back this data as part of the questions-list response from the backend?

Reply:
@vinitamurthi, I have moved the `TopicsAndSkillsDashboardApiService` to the  `topicQuestionsTab` directive, adding the `categorisedSkills` to questions-list response seems a bit odd to me because `categorisedSkills` has data from all the topics.
But still, if there is some specific reason which I missed, I can move it to the response data.